features/FxA-15-OAuth-Management/README.md

```diff
@@ -0,0 +1,37 @@
+Disconnect OAuth services from FxA
+======================
+
+https://mozilla.aha.io/features/FXA-15
+
+## Problem Summary
+
+As a Firefox Accounts user I want to be able to revoke access to the
+services that I logged into.  
+
+****
+
+## Outcomes
+
+## Hypothesis
+
+## Metrics
+
+****
+
+## Detailed design
+
+![](https://www.lucidchart.com/publicSegments/view/bcc7ac8f-cb6d-44cd-a5ca-b59a786168ca/image.jpeg?w=1)
+
+[Edit Mockup](https://www.lucidchart.com/documents/edit/bdafa87d-ea17-4beb-8381-f6da19d7359a#)
+
+The initial session management dashboard would
+revoke the non-expired tokens using the web UI.
+It will be displayed to logged in users of FxA
+in the settings view.
+
+It will also list the sessions that were created
```

Comment:
Does `sessions` here mean `OAuth tokens`? If so, we are going to have to be a whole lot more strict about deleting OAuth tokens on the client side, or giving them a short expiration as @rfk mentioned for another bug. We create OAuth codes/tokens in several places and leave many of them hanging around. I'm unsure how the browser deals with this.

One question that comes out of this, say I revoke a token for the browser. The browser could just then request a new one. Would revoking access to the browser, or FxA for that matter, prevent new tokens from being created?